### ISSUE TITLE
@apply doesn't work inside imported files.

### ISSUE THREAD

**Issue by fauzan121002** [windicss/nuxt-windicss-module#100]
**Describe the bug**
@apply will compiled on Global CSS which if directed to a stand-alone CSS file, but it will not work on files that import other files (wrapper file).

**To Reproduce**
1. Setup nuxt-windicss just like the docs instruction.
2. Define wrapper file for global CSS.
```js
css: ['~/assets/css/windi.css'],
```
Inside windi.css
```css
@import './navbar.css';
@import './footer.css';
@import './themes.css';
@import './effects.css';
```

**Expected behavior**
I expect @apply would work / compiled in the others files but the result was @apply doesn't work / compiled when others imported css except @apply are working.

**Screenshots**
![Capture](https://user-images.githubusercontent.com/50759463/122377918-b2aeb880-cf8f-11eb-939c-59589af5c277.PNG)

**Additional context**
@apply **works** if we import the files like this
```js
css: [
  '~/assets/css/navbar.css',
  '~/assets/css/footer.css',
  '~/assets/css/themes.css',
  '~/assets/css/effects.css',
],
```

**Comment by harlan-zw** [comment]
Hey @fauzan121002 

Thanks for reporting this issue. Should be fixed in the latest release - 1.1.0

**Comment by fauzan121002** [comment]
> Hey @fauzan121002
> 
> Thanks for reporting this issue. Should be fixed in the latest release - 1.1.0

Thanks @harlan-zw
